server/requirements_streamlit.txt

```diff
@@ -1,3 +1,4 @@
+mongomock==4.3.0
```

Comment:
Should we create a project level requirements file for running the tests and coverage?

Reply:
Good point, created an issue for that (375)